stoobly_agent/app/cli/helpers/endpoints_import_service.py

```diff
@@ -92,7 +103,12 @@ def import_query_param_name(project_id: int, endpoint_id: int, query_param_name_
 
     res.raise_for_status()
 
-def import_response_param_name(project_id: int, endpoint_id: int, response_param_name_resource: ResponseParamNamesResource, response_param_name: RequestComponentName):
+def import_response_param_name(project_id: int, endpoint_id: int, response_param_name_resource: ResponseParamNamesResource, response_param_name: RequestComponentName, parents: Dict[int, int]):
+    response_param_name_id = None
+    parent_id = response_param_name['response_param_name_id']
```

Comment:
```suggestion
    parent_id = response_param_name.get('response_param_name_id')
```

In case `response_param_name_id` does not exist, won't throw an error